### ISSUE TITLE
Keyboard navigation when using multi row layout differs between Angular and WC

### ISSUE THREAD

**Issue by skrustev** [IgniteUI/igniteui-angular#13566]
## Description  
The behavior when using the keyboard navigation to navigate between cells when multi row layout is applied differs between Angular and Web Components.

 * igniteui-angular version: 16.0.x
 * browser: any

## Steps to reproduce  

1. Open https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/multi-row-layout#angular-multi-row-layout-example
2. Navigate between cells using Arrow keys
3. Open https://www.infragistics.com/products/ignite-ui-web-components/web-components/components/grids/grid/multi-row-layout#web-components-multi-row-layout-example
4. Navigate between cells using Arrow keys

## Result  
The experience using the keyboard navigation is different between Angular and Web Components.

## Expected result  
The experience using the keyboard navigation between Angular and Web Components should be the same.
